config/phenomenal_tools2container.yaml

```diff
@@ -443,6 +443,13 @@ assignment:
   max_pod_retrials: 1
   tools_id:
    - pathway_enrichment
+- docker_repo_override: container-registry.phenomenal-h2020.eu
+  docker_owner_override: phnmnl
+  docker_image_override: fingerprintclustering
+  docker_tag_override: latest
```

Comment:
Please fix the container version with the relevant container tag generated from Jenkins?